### PR TITLE
[EAGLE-982] The log length has exceeded the limit of 4 MB in Travis

### PR DESCRIPTION
**Pull request by asdf2014**
Maven throw too many warnings for checking `Code Style` in `Travis`. Should we disable it for a while until we fix those `Code Style` issues?
```
The log length has exceeded the limit of 4 MB (this usually means that the test suite is raising the same exception over and over).
The job has been terminated
```

(https://issues.apache.org/jira/browse/EAGLE-982)

### CODE REVIEW

File: eagle-core/eagle-query/eagle-entity-base/src/main/java/org/apache/eagle/log/base/taggedlog/RowkeyAPIEntity.java [hunk 1]
```diff
@@ -16,66 +16,79 @@
  */
```
Comment (jhsenjaliya) [inline eagle-core/eagle-query/eagle-entity-base/src/main/java/org/apache/eagle/log/base/taggedlog/RowkeyAPIEntity.java]: pls remove this indentation changes, its not related to the issue

**Comment by jhsenjaliya** [comment]
@asdf2014 , can u please squash the commits ?

File: eagle-core/eagle-query/eagle-entity-base/src/main/java/org/apache/eagle/log/base/taggedlog/RowkeyAPIEntity.java (hunk 1 above)
Comment (asdf2014) [inline eagle-core/eagle-query/eagle-entity-base/src/main/java/org/apache/eagle/log/base/taggedlog/RowkeyAPIEntity.java]: Okey.

**Comment by asdf2014** [comment]
@jhsenjaliya Already done.

**Comment by asdf2014** [comment]
Those changes about `.travis.yml` file shouldn't effect `Jenkins`. It seems like the `maven-surefire-plugin` throw a `ExecutionException` to make this build process failed.
```bash
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.19.1:test (default-test) on project eagle-app-streamproxy: ExecutionException The forked VM terminated without properly saying goodbye. VM crash or System.exit called?
[ERROR] Command was /bin/sh -c cd /home/jenkins/jenkins-slave/workspace/incubator-eagle-pr-reviewer/eagle-core/eagle-app/eagle-app-streamproxy && /usr/local/asfpackages/java/jdk1.8.0_121/jre/bin/java -XX:-UseGCOverheadLimit -jar /home/jenkins/jenkins-slave/workspace/incubator-eagle-pr-reviewer/eagle-core/eagle-app/eagle-app-streamproxy/target/surefire/surefirebooter8929133803599728653.jar /home/jenkins/jenkins-slave/workspace/incubator-eagle-pr-reviewer/eagle-core/eagle-app/eagle-app-streamproxy/target/surefire/surefire4241285850518905830tmp /home/jenkins/jenkins-slave/workspace/incubator-eagle-pr-reviewer/eagle-core/eagle-app/eagle-app-streamproxy/target/surefire/surefire_2332892144259826349419tmp
```

**Comment by jhsenjaliya** [comment]
@asdf2014 please squash the commits again and I should merge this.

**Pull request closed without merging**